app/controllers/restaurant_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantController < ApplicationController
+
+  def show
+    service = RevItUpService.new(current_user.lat, current_user.lng, meters_to_miles, params["price"])
```

Comment:
Refactor: We could have the service initialized within the facade to push this logic out of the view.